Source/WebCore/platform/mediastream/gstreamer/GStreamerIncomingTrackProcessor.cpp

```diff
@@ -71,13 +71,13 @@ void GStreamerIncomingTrackProcessor::configure(ThreadSafeWeakPtr<GStreamerMedia
     if (auto ssrc = gstStructureGet<unsigned>(structure, "ssrc"_s)) {
         m_data.ssrc = *ssrc;
         auto msIdAttributeName = makeString("ssrc-"_s, *ssrc, "-msid"_s);
-        if (auto msIdAttribute = gstStructureGetString(structure, msIdAttributeName)) {
-            auto components = msIdAttribute.toStringWithoutCopying().split(' ');
+        if (auto msIdAttribute = gstStructureGetString(structure, CStringView::fromString(msIdAttributeName))) {
```

Comment:
This looks suspicious too... afaik the msid attribute should be treated as UTF-8